### ISSUE TITLE
Link to image alt text guide is not accessible

### ISSUE THREAD

**Issue by Alvant** [arXiv/arxiv-docs#969]
Page: [LaTeX Markup Best Practices for Successful HTML Papers](https://info.arxiv.org/help/submit_latex_best_practices.html) ([gh](https://github.com/arXiv/arxiv-docs/blob/develop/source/help/submit_latex_best_practices.md)).

Section: Alt text for images

> Without alt text, images are essentially invisible to screen readers. Additionally, alt text improves discoverability of your content within full-text searches. Dr. Cynthia Bennett has written [this excellent guide](https://docs.google.com/document/d/1P62YxEDDfG4DAofnaPiO4Y665eK4rCCD_EwjtF87iBk/edit?usp=sharing) to image descriptions and alt text.

If try to click the link to the guide:

![image](https://github.com/user-attachments/assets/81c91594-c6ee-4427-a4a4-b028c6be213f)

(It says "No Access")

Expected: the promised guide.

**Comment by alisonhofer** [comment]
@SBBCornell I am wondering if we ever had a copy of the guide and shared it or if this was Dr. Cynthia Bennett's copy that was linked?
